tests/admin/test_api.py

```diff
@@ -15,6 +17,43 @@ def admin_api() -> FlaskClient:
     return application.test_client()
 
 
+def test_list_migration_status(admin_api: FlaskClient) -> None:
+    response = admin_api.get("/migrations/system/list")
+    assert response.status_code == 200
+    expected_json = [
+        {"blocking": False, "migration_id": "0001_migrations", "status": "completed"}
+    ]
+    assert json.loads(response.data) == expected_json
+
+    # invalid migration group
+    response = admin_api.get("/migrations/bad_group/list")
+    assert response.status_code == 400
+
+    response = admin_api.get("/migrations/sessions/list")
+    assert response.status_code == 200
+    expected_json = [
+        {"blocking": False, "migration_id": "0001_sessions", "status": "completed"},
+        {
+            "blocking": False,
+            "migration_id": "0002_sessions_aggregates",
+            "status": "completed",
+        },
+        {
+            "blocking": False,
+            "migration_id": "0003_sessions_matview",
+            "status": "completed",
+        },
+    ]
+
+    def sort_by_migration_id(migration: Any) -> Any:
+        return migration["migration_id"]
```

Comment:
How come sorting by migration ID? Although by convention we use incrementing numbers for migration IDs, this is just a convention and the ordering is defined explicitly by the migration group.

Reply:
This was just to compare the lists in the assert.